src/ansys/aedt/core/desktop.py

```diff
@@ -1904,14 +1900,12 @@ def submit_job(
                 version_name = "v" + version.replace(".", "")
             else:
                 version_name = "AnsysEM" + version
-            if os.path.exists(r"\\" + clustername + r"\AnsysEM\{}\Win64\ansysedt.exe".format(version_name)):
-                aedt_full_exe_path = (
-                    r"\\\\\\\\" + clustername + r"\\\\AnsysEM\\\\{}\\\\Win64\\\\ansysedt.exe".format(version_name)
-                )
-            elif os.path.exists(r"\\" + clustername + r"\AnsysEM\{}\Linux64\ansysedt".format(version_name)):
+            if os.path.exists(r"\\" + clustername + f"\\AnsysEM\\{version_name}\\Win64\\ansysedt.exe"):
                 aedt_full_exe_path = (
-                    r"\\\\\\\\" + clustername + r"\\\\AnsysEM\\\\{}\\\\Linux64\\\\ansysedt".format(version_name)
+                    r"\\\\\\\\" + clustername + f"\\\\AnsysEM\\\\{version_name}\\\\Win64\\\\ansysedt.exe"
                 )
+            elif os.path.exists(r"\\" + clustername + f"\AnsysEM\{version_name}\Linux64\ansysedt"):
+                aedt_full_exe_path = r"\\\\\\\\" + clustername + f"\\\\AnsysEM\\\\{version_name}\\\\Linux64\\\\ansysedt"
```

Comment:
This could be rewritten with `chr(92)` or `"\\"` and using f-strings instead of str addition.

Reply:
@Samuelopez-ansys @maxcapodi78 Can you let me know if the way we are handling `'\'` caracters is correct here ? I'm confused by the fact of testing `"\\"` then assigning `"\\\\\\\\"` :/
Any reason we should keep to the current code ? Is it even used in the first place ?

Reply:
@gmalinve Why did you change Linux64 to Win64?

I think we need to keep all the // because the .areg file has this kind of syntax. But maybe I am wrong.

Reply:
@Samuelopez-ansys I didn't. Go to the file and compare it. Nothing has changed if not the f-strings